lib/active_model/serializer.rb

```diff
@@ -28,8 +28,8 @@ def self.inherited(base)
       base._attributes_keys = self._attributes_keys.try(:dup) || {}
       base._associations = self._associations.try(:dup) || {}
       base._urls = []
-      serializer_file = File.open(caller.first[/^[^:]+/])
-      base._cache_digest = Digest::MD5.hexdigest(serializer_file.read)
+      serializer_file_path = caller.first[/\A\S+(?=:\d+:in)/]
+      base._cache_digest = Digest::MD5.hexdigest(File.read(serializer_file_path))
     end
```

Comment:
> if you really care about Windows user

Of course I care.  No reason not add a regression test

Also, I'm not a collab so I can't merge it sorry :)

@kurko @joaomdmoura Let's see if we can set up an [appveyor](http://www.appveyor.com/) account to test on windows.  I know [RSpec does this](https://github.com/rspec/rspec-support/blob/master/appveyor.yml) and it works pretty well.